features/dd-sdk-android-rum/src/main/kotlin/com/datadog/android/rum/internal/metric/slowframes/UISlownessMetricDispatcher.kt

```diff
@@ -0,0 +1,151 @@
+/*
+ * Unless explicitly stated otherwise all files in this repository are licensed under the Apache License Version 2.0.
+ * This product includes software developed at Datadog (https://www.datadoghq.com/).
+ * Copyright 2016-Present Datadog, Inc.
+ */
+package com.datadog.android.rum.internal.metric.slowframes
+
+import androidx.annotation.MainThread
+import androidx.annotation.WorkerThread
+import com.datadog.android.api.InternalLogger
+import com.datadog.android.api.InternalLogger.Target
+import com.datadog.android.rum.configuration.SlowFramesConfiguration
+import com.datadog.tools.annotation.NoOpImplementation
+import java.util.concurrent.ConcurrentHashMap
+import java.util.concurrent.atomic.AtomicInteger
+
+@NoOpImplementation
+internal interface UISlownessMetricDispatcher {
+
+    fun onViewCreated(viewId: String)
+
+    fun incSlowFrame(viewId: String)
+
+    fun incIgnoredFrame(viewId: String)
+
+    fun incFreezeFrame(viewId: String)
```

Comment:
nit: maybe we can use `increment` instead of `inc` and add `count`? making it like `incrementSlowFrameCount`, `incrementIgnoredFrameCount`, etc.